### PR TITLE
package.json: specify files to include in distribution

### PR DESCRIPTION
**Pull request by davidchambers**
I'd like to release 1.0.0. It'd be nice to remove cruft from the tarball.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,5 +15,12 @@
     "mocha": "1.18.x",
     "uglify-js": "2.4.x",
     "xyz": "0.5.x"
-  }
+  },
+  "files": [
+    "/LICENSE",
+    "/README.md",
+    "/base64.js",
+    "/base64.min.js",
```
Comment (davidchambers) [inline package.json]: Shall we include this? I don't know whether there's a good reason to do so.

Comment (davidchambers) [inline package.json]: Okay. I've removed the minified file from the list.

**Comment by michaelficarra** [comment]
From the npm documentation,

> Certain files are always included, regardless of settings:
> 
> * `package.json`
> * `README` (and its variants)
> * `CHANGELOG` (and its variants)
> * `LICENSE` / `LICENCE`

So this PR can be simplified a bunch.

**Comment by davidchambers** [comment]
We could rely on npm's defaults, but I appreciate the straightforwardness of the current approach. It's clear to any reader which files are intended to be included in the distribution. Were we to rely on npm's defaults this would not be the case. The reader would need to:

1.  consult the documentation to determine the default set of filenames;
2.  consult the repository to determine which files it contains;
3.  determine the intersection of these two sets; and
4.  determine the union of the resulting set and the set of filenames listed in __package.json__.

I'm happy to pay the price of three extra lines in a JSON file, in this case. :)

**Comment by michaelficarra** [comment]
Works for me.

**Comment by davidchambers** [comment]
I just published [1.0.0][1]. It's not often one gets to run `make release-major`. :)


[1]: https://github.com/davidchambers/Base64.js/releases/tag/1.0.0

**Pull request merged**